envs/hanabi_env.py

```diff
@@ -77,6 +77,7 @@ def __init__(self, num_envs, gpu_id, debug_compile=True, config=None):
 
         # sim = None
         sim = hanabi_python.HanabiSimulator(
+            exec_mode = hanabi_python.ExecMode.CUDA,
```

Comment:
This needs to be configurable, but I wasn't familiar with your python config setup. ExecMode.CPU for the cpu version.
Additionally, the tensors reside on the CPU so need to be copied to the GPU.